captum/optim/_param/image/transform.py

```diff
@@ -1,11 +1,9 @@
-import logging
 import math
 import numbers
 
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
-from kornia.geometry.transform import rotate, scale, shear, translate
 
 device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
Do we need this ?

Reply:
Nope, we no longer need Kornia for anything! The rotate and shear transforms from Torchvision can be used instead, and thus I removed some of the Kornia replacement code I wrote. The translation and scale transforms technically exist in Torchvision as well under RandomAffine, but they are not setup ideally for what we need so I kept those versions.

Reply:
The two example notebooks showcase examples of using both Torchvision & Captum.optim transforms at the same time.

Reply:
Awesome! Thank you for do that!